app/views/accompaniment_leader/activities/_accompaniment_modal.html.erb

```diff
@@ -41,18 +41,20 @@
               <%= render 'shared/scoped_activity_users_details', activity: activity, scope: 'volunteer' %>
             <% end %>
             <% if activity.accompaniment_reports.present? %>
-              <strong>Accompaniment Leader Notes:</strong>
+              <strong>Report Back:</strong>
               <ol>
                 <% activity.accompaniment_reports.each do |report| %>
-                  <li><strong>Notes by <%= report.user.name if report.user.present? %>:  </strong><%= report.notes %></li>
+                  <li><strong>Report Back by: </strong><%= report.user.name if report.user.present? %>
```

Comment:
@latazzajones I love the new format with the divider lines that you added for the admin views, like app/views/admin/activities/_activity.html.erb
Could we add that formatting here as well?

Reply:
Done! These accompaniment models could potentially be moved out into a partial but I think that's part of our discussion around refactoring our views and managing authorization.